webapp/channels/src/components/no_results_indicator/no_results_indicator.test.tsx

```diff
@@ -1,109 +1,133 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow} from 'enzyme';
 import React from 'react';
 
 import NoResultsIndicator from 'components/no_results_indicator/no_results_indicator';
 
+import {withIntl} from 'tests/helpers/intl-test-helper';
+import {render, screen} from 'tests/react_testing_utils';
+
 import {NoResultsVariant, NoResultsLayout} from './types';
 
 describe('components/no_results_indicator', () => {
-    test('should match snapshot with default props', () => {
-        const wrapper = shallow(
-            <NoResultsIndicator/>,
-        );
-
-        expect(wrapper).toMatchSnapshot();
-    });
-
-    test('should match snapshot with variant ChannelSearch', () => {
-        const wrapper = shallow(
-            <NoResultsIndicator
-                iconGraphic={<div>{'Test'}</div>}
-                variant={NoResultsVariant.ChannelSearch}
-            />,
+    test('should render with variant ChannelSearch', () => {
+        render(
+            withIntl(
+                <NoResultsIndicator
+                    iconGraphic={<div>{'Test'}</div>}
+                    variant={NoResultsVariant.ChannelSearch}
+                    titleValues={{channelName: 'test-channel'}}
+                />,
+            ),
         );
 
-        expect(wrapper).toMatchSnapshot();
+        expect(screen.getByText('Test')).toBeInTheDocument();
+        expect(screen.getByText(/No results for/)).toBeInTheDocument();
     });
 
-    test('should match snapshot with variant Mentions', () => {
-        const wrapper = shallow(
-            <NoResultsIndicator
-                iconGraphic={<div>{'Test'}</div>}
-                variant={NoResultsVariant.Mentions}
-            />,
+    test('should render with variant Mentions', () => {
+        render(
+            withIntl(
+                <NoResultsIndicator
+                    iconGraphic={<div>{'Test'}</div>}
+                    variant={NoResultsVariant.Mentions}
+                />,
+            ),
         );
 
-        expect(wrapper).toMatchSnapshot();
+        expect(screen.getByText('Test')).toBeInTheDocument();
+        expect(screen.getByText('No mentions yet')).toBeInTheDocument();
     });
 
-    test('should match snapshot with variant FlaggedPosts', () => {
-        const wrapper = shallow(
-            <NoResultsIndicator
-                iconGraphic={<div>{'Test'}</div>}
-                variant={NoResultsVariant.FlaggedPosts}
-            />,
+    test('should render with variant FlaggedPosts', () => {
```

Comment:
Should we add some assertions for the button? (or whatever we are passing the `subtitleValues` for)